### ISSUE TITLE
Orientation of steppers

### ISSUE THREAD

**Issue by mskogly** [dpenney/grbl-1.1-polr#3]
Should the shaft of the steppers point towards the wall, our out from the wall?

**Comment by dpenney** [comment]
You can reverse the direction of the steppers via the grbl settings so it
shouldn't matter, in my case they pointed away.

**Comment by mskogly** [comment]
Ok thanks, will test.

**Comment by mskogly** [comment]
Hm, still as confused :) Tried orienting the steppers with the shafts pointing out, tried reversing, and tried flipping the cables connected to the cnc-shield, but couldn´t really make sense of it. I saw in another thread that it might be related to the homing point, which is placed in the middle of the canvas on the x axis (but doesnt matter where on the y axis?). You wouldn´t happen to have any detatailed build instructions? Would be great to have something to duplicate both physically and with the settings, simply to reduce the numbers of uncertainties . 

(Actually got so frustrated with myself that I dusted off my ancient polargraph yesterday. https://pappmaskin.no/2016/08/pen-plotter-1/ . It still works fine, but not a big fan of the polargraph controller which is why I wanted to use grbl instead).

**Comment by dpenney** [comment]
Yeah your zero point is the center of the canvas. When you are centered a
positive y movement should result in your left stepper going anti clockwise
and your right stepper moving clockwise shortening both cables and raising
your pointer. If they don't do that you need to edit the direction mask in
grbl accordingly.

so g1 x0 y100 should go straight up, steppers turning opposite directions.

On machine startup it expects to be at 0,0. So start it and then manually
move your pointer to that center spot.

Damian
